drivers/led/issi/is31fl3741.h

```diff
@@ -81,11 +82,11 @@
 #endif
 
 typedef struct is31fl3741_led_t {
-    uint32_t driver : 2;
-    uint32_t r : 10;
-    uint32_t g : 10;
-    uint32_t b : 10;
-} __attribute__((packed)) is31fl3741_led_t;
+    uint8_t  driver : 2;
+    uint16_t r : 9;
```

Comment:
`10` to `9` is because max LEDs is 351, and 9 is minimum bits to represent?

Reply:
Yep. The PWM registers on this driver are split across two pages:
![image](https://github.com/qmk/qmk_firmware/assets/4781841/8bce5c7c-c9e0-4ba3-9067-abfda676dcb3)
I will probably renumber the `CSx_SWy` defines to more closely resemble the register addresses, with `0x100` denoting page 1, which works out to the same number of bits anyway.